src/commands/analyze/index.ts

```diff
@@ -193,7 +193,7 @@ export default class Analyze extends Command {
       configFile,
       networks,
       debug: this.debug,
-      processTransactions: this.processTransactions,
+      // processTransactions: this.processTransactions, // TODO: work on it
```

Comment:
Do we need to create a new ticket for this @sogoiii ?

Reply:
@alexanderattar I asked @BravoNatalie to disable the analyze flag for the time being.